backend/tests/test_accounts.py

```diff
@@ -0,0 +1,185 @@
+import pytest
```

Comment:
Need a test for updating a valid account since you have implemented that functionality